source/SAMRAI/tbox/ExecutionPolicy.h

```diff
@@ -111,6 +111,11 @@ struct policy_traits<policy::parallel> {
    >;
 
    using ReductionPolicy = RAJA::cuda_reduce;
+
+   using WorkGroupPolicy = RAJA::WorkGroupPolicy<
+      RAJA::cuda_work_async<1024>, 
```

Comment:
Can you provide a way to set the workgroup block size in case people run into cuda linking issues?